src/compute-client/src/controller.rs

```diff
@@ -210,9 +210,9 @@ impl ComputeReplicaLocation {
 #[derive(Copy, Clone, Debug, Serialize, Deserialize, Eq, PartialEq)]
 pub struct ComputeReplicaAllocation {
     /// The memory limit for each process in the replica.
-    pub memory_limit: Option<MemoryLimit>,
+    pub memory_limit: MemoryLimit,
     /// The CPU limit for each process in the replica.
-    pub cpu_limit: Option<CpuLimit>,
```

Comment:
Do we not write this down in the stash? Let's make sure there are no incompatible changes.

Reply:
I checked the code, and it seems we only write the T-shirt size. @jkosh44 or someone else on the adapter team, can you confirm?

Reply:
Joe showed me where I can see the schema of all the stash collections in storage.rs, and I verified that there's nowhere the concrete size (as opposed to the T-shirt size) gets stored.